assets/pay-stamp-duty-land-tax.css

```diff
@@ -0,0 +1,13 @@
+.secondary-text {
```

Comment:
Any chance we can rename this file to just be content-history-prototype.css so we can put all our tweaks in it?

Off top of my head I can't think of any changes we're doing that won't apply to all 3 things.